Model-Docs/example-model.yaml

```diff
@@ -103,6 +187,9 @@ parameters:
       # expects
       type: NumberParameter
     default: 1
+    #minimum/maximum values for the parameter [OPTIONAL]
+    minumum: 0
```

Comment:
Could we put `min`/`max` in the `metadata` field?

Reply:
Does the DCAT accept parameters? If they are for the model catalog, we don't have a metadata field, so it's just all metadata... In any case, we can leave it for consistency. It's just that I don't like the metadata field much, to be honest.

Reply:
No model parameters in DCAT, and I agree about disliking `metadata` since everything here really is metadata. That said it is nice to have a specific section to store arbitrary information.

Reply:
Maybe we can keep for this for files but remove it for everything else? 
I think min/max should not be in metadata because it's really important for validation and correct setup